src/games/strategy/util/Triple.java

```diff
@@ -14,7 +14,7 @@
     return new Triple(first,second,third);
   }
 
-  private Triple(final F first, final S second, final T third) {
+  public Triple(final F first, final S second, final T third) {
```

Comment:
No need to respond to a comment with anything other than a code commit to address it, and/or a response indicating that you are doing otherwise. When you update the code the comments on it will be marked as out of date